bot_management/bot_manager.py

```diff
@@ -0,0 +1,32 @@
+import uuid
+from bot_management.bot_body import Bot
+import threading
+from bot_management.my_keys import *
+
+my_array = []
```

Comment:
What does `my_array` mean? I know this is for bot storage but why this is a somehow 'global' variable not specific for BotManager class/object?

Reply:
Because it worked just as i expected it to work when i put it here and didn,t work when put in other place